bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -340,10 +340,11 @@ private String formatReviewer(HostUser reviewer) {
         }
     }
 
-    private String getChecksList(PullRequestCheckIssueVisitor visitor) {
-        return visitor.getChecks().entrySet().stream()
-                      .map(entry -> "- [" + (entry.getValue() ? "x" : " ") + "] " + entry.getKey())
-                      .collect(Collectors.joining("\n"));
+    private String getChecksList(PullRequestCheckIssueVisitor visitor, boolean isCleanBackport) {
+        var checks = isCleanBackport ? visitor.getReadyForReviewChecks() : visitor.getChecks();
```

Comment:
I find the name of the new method a little confusing. A clean backport is ready for integration not for review. Maybe just `getReadyChecks` or `getChecksClean`? I'll leave it up to you.

Reply:
I understand the confusion. I chose the name to fit in with the current model and naming. When CheckRun evaluates if a cleanBackport PR is ready for integration, it calls PullRequestCheckIssueVisitor.isReadyForReview() (instead of .messages().isEmpty()). So the state we are checking for in the visitor is already named "readyForReview". The new method builds on that and returns the checks that were used to decide if it was readyForReview.

Reply:
I should add that the CheckRun knows about clean backports, while the visitor does not, it just evaluates Jcheck checks and reports on the outcome.

Reply:
OK, thanks for the explanation.